### DISCUSSION TITLE
Video sobre metodologias

### DISCUSSION THREAD

**Discussion by AlejandroDeheza** [dds-jv/foro#275] [Consultas generales] [answered]
Buenas, una consulta, puede ser que el video sobre metodologias no este visible en la bitacora o en el campus virtual?  
soy del curso de los viernes a la noche

**Comment by roli93** [comment, marked as answer]
Hola! Lo vamos a estar subiendo en estos días. De todas formas no hace falta que lo vean necesariamente para este viernes

**Reply by AlejandroDeheza** [reply]
Gracias!
